recommendation/tests.py

```diff
@@ -1,6 +1,8 @@
 from django.test import TestCase
 from django.urls import reverse
 from django.utils import timezone
+from django.utils.http import urlsafe_base64_encode
```

Comment:
base_64 hash is kind of easy to crack. Is there any better hashing technique that can be used from the same library?

Reply:
In that specific library the other one would be base_32 . would you have any recommendations for better hashing?

Reply:
I am not aware of any other library. We can use this for now.